src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -63,11 +63,13 @@ public static class DataTestUtility
         public static string AADUserIdentityAccessToken = null;
         public const string UdtTestDbName = "UdtTestDb";
         public const string AKVKeyName = "TestSqlClientAzureKeyVaultProvider";
-
+        public const string EventSourcePrefix = "Microsoft.Data.SqlClient";
+        public const string MDSEventSourceName = "Microsoft.Data.SqlClient.EventSource";
+        public const string AKVEventSourceName = "Microsoft.Data.SqlClient.AlwaysEncrypted.AzureKeyVaultProvider.EventSource";
         private const string ManagedNetworkingAppContextSwitch = "Switch.Microsoft.Data.SqlClient.UseManagedNetworkingOnWindows";
 
         private static Dictionary<string, bool> AvailableDatabases;
-        private static TraceEventListener TraceListener;
+        public static BaseEventListener TraceListener;
```

Comment:
It's the TraceListener we enable traces from, it may not need to be public anymore so I can change it back to private.
```suggestion
        private static BaseEventListener TraceListener;
```